src/lib/plugin.js

```diff
@@ -414,8 +414,8 @@ module.exports = class PluginPaymentChannel extends EventEmitter2 {
     }
   }
 
-  _getLowestBalance () {
-    return Promise.resolve(this._lowestBalance.get())
+  async _handleGetBalance () {
+    return await this._transfers.getBalance()
```

Comment:
eslint is not up to date. This should complain, see: http://eslint.org/docs/rules/no-return-await